src/resources/lang/en/base.php

```diff
@@ -21,14 +21,28 @@
     'name'                 => 'Name',
     'email_address'        => 'E-Mail Address',
     'password'             => 'Password',
+    'old_password'         => 'Old Password',
+    'new_password'         => 'New Password',
     'confirm_password'     => 'Confirm Password',
     'remember_me'          => 'Remember Me',
     'forgot_your_password' => 'Forgot Your Password?',
     'reset_password'       => 'Reset Password',
     'send_reset_link'      => 'Send Password Reset Link',
     'click_here_to_reset'  => 'Click here to reset your password',
+    'change_password'      => 'Change Password',
     'unauthorized'         => 'Unauthorized.',
     'dashboard'            => 'Dashboard',
     'handcrafted_by'       => 'Handcrafted by',
     'powered_by'           => 'Powered by',
+    'my_account'           => 'My Account',
+    'update_account_info'  => 'Update Account Info',
+    'save'                 => 'Save',
+    'cancel'               => 'Cancel',
+    'error'                => 'Error',
+    'success'              => 'Success',
+    'old_password_wrong'   => 'Old password is incorrect.',
```

Comment:
i guess old_password_incorrect would be better according to its value. At least for searching keys

Reply:
So long as don't get a `T_PAAMAYIM_NEKUDOTAYIM` silliness :P